demo/demo-server-servlet/src/main/webapp/WEB-INF/web.xml

```diff
@@ -27,17 +27,30 @@
 		</param-value>
 	</context-param>
 
+	<!-- usage 1 equals usage 2 -->
```

Comment:
document had not finished now, so remain the information here
after document finished, move the description to the document